README.md

```diff
@@ -43,61 +36,70 @@ If you are not familiar with these notions, we will first recommend you get more
   - install [Google Cloud SDK](https://cloud.google.com/sdk/docs/downloads-interactive).
   - authenticate my SDK account by running `gcloud auth application-default login` in the terminal.
 
-2. and GSVA for ssGSEA in R `R` run `R -e 'if(!requireNamespace("BiocManager", quietly = TRUE)){install.packages("BiocManager")};BiocManager::install(c("GSEABase", "erccdashboard", "GSVA", "DESeq2"));`
+2. and GSVA for ssGSEA in R `R` run `R -e 'if(!requireNamespace("BiocManager", quietly = TRUE)){install.packages("BiocManager")};BiocManager::install(c("GSEABase", "erccdashboard", "GSVA", "DESeq2"));'`
 
 3. For Python use the requirements.txt file `pip install -r requirements.txt` 
 
-## For Broad People
+## For Internal Users <a name="internal-users"></a>
 
-> Mind that you need to know about the tools listed above
-- Ask dbgap access to your supervisor
+> To learn about the tools we use in the pipeline, see [here](#running-pipeline) for a detailed walkthrough
 
 ### Getting Terra Access
 
 1. You will need to request access to the following terra workspaces:
   - https://app.terra.bio/#workspaces/broad-firecloud-ccle/DepMap_Mutation_Calling_CGA_pipeline
   - https://app.terra.bio/#workspaces/broad-firecloud-ccle/DepMap_hg38_RNAseq
   - https://app.terra.bio/#workspaces/broad-firecloud-ccle/DepMap_WES_CN_hg38
+  - https://app.terra.bio/#workspaces/broad-firecloud-ccle/DepMap_WGS_CN
+
 The current owners of these workspaces should give you access.
-2. For the mutation pipeline you will also need to request DBGap access (required for TCGA workflows): https://docs.google.com/document/d/1hwsjUypqUpse7IeMyBLKEXmdlXUzfBa4e4p9teCVtaw/edit?usp=sharing
-3. Ask Sarah Young for access to required billing projects (e.g. broad-firecloud-ccle)
+
+2. For the mutation pipeline you will also need to request dbGaP access (required for TCGA workflows). See CCLE/new hiree section on Asana for details.
+3. Aquire access to required billing projects (e.g. broad-firecloud-ccle). See CCLE/new hiree section on Asana for details.
 4. Get access to the following Terra groups:
   - DEPMAP_CCLE_DATA
   - DEPMAP-PIPELINES
   - CCLE2-DATA
   - CCLE-PIPELINE
-5. If you need to get access to the depmap data:
-  - __Exome__ [IBM](https://firecloud.terra.bio/#workspaces/broad-genomics-delivery/Getz_IBM_CellLines_Exomes/data)
-  - __Exome__ [Broad](https://firecloud.terra.bio/#workspaces/broad-firecloud-ccle/CCLE_DepMap_WES)
-  - __Exome__ [Broad](https://firecloud.terra.bio/#workspaces/broad-genomics-delivery/CCLE_DepMap_WES)
-  - __rna__ [IBM](https://firecloud.terra.bio/#workspaces/broad-genomics-delivery/Getz_IBM_CellLines_RNASeqData/data)
-  - __rna__ [Broad](https://firecloud.terra.bio/#workspaces/broad-firecloud-ccle/CCLE_DepMap_RNAseq)
-  - __rna__ [Broad](https://firecloud.terra.bio/#workspaces/broad-genomics-delivery/CCLE_DepMap_RNAseq)
-6. Request access to the data bucket `gs://ccle_bams/`
+5. If you need to get access to the depmap data, use the following links:
```

Comment:
there is not the depmap data here. this is where the data is sent to us from GP. it gets erased at some point. I think Exome might soon not contain anything anymore..

Reply:
should the WES links be removed now or should we wait?